### ISSUE TITLE
Remoteconfig crashes on android

### ISSUE THREAD

**Issue by moleki** [firebase/firebase-cpp-sdk#264]
### [REQUIRED] Please fill in the following fields:

  * Firebase C++ SDK version: 6.16.1
  * Problematic Firebase Component: RemoteConfig (Auth, Database, etc.)
  * Other Firebase Components in use: Analytics, Messaging (Auth, Database, etc.)
  * Platform you are using the C++ SDK on: Mac (Mac, Windows, or Linux)
  * Platform you are targeting: Android (iOS, Android, and/or desktop)

### [REQUIRED] Please describe the issue here:

I am getting a crash in native c++ on android with the latest version of firebase c++ sdk 6.16.1 and firebase sdk remoteconfig 20.0.2.
If I downgrade remote config to 19.2.0, it works fine.

Here is the crash log

> Unable to find Method com/google/firebase/remoteconfig/FirebaseRemoteConfig.activateFetched (signature '()Z', instance). Please verify the AAR which contains the com/google/firebase/remoteconfig/FirebaseRemoteConfig class is included in your app.

Remoteconfig C++ doesn't seem to support ActivateFetched() which has been deprecated. (cf [https://stackoverflow.com/questions/57494893/firebaseremoteconfig-activatefetched-is-deprecated-what-to-use-instead](url)
Do you guys have an update or fix soon because this prevent us from upgrading firebase to the latest version?

**Comment by patm1987** [comment]
Hi @moleki ,

Between the 6.x and 7.x branches of the C++ SDK, there have been some significant changes to the underlying RemoteConfig SDK. If you want to stay at the 6.x branch of the C++ SDK you will need to use an older version of the associated Android library, but I'd recommend updating to the [recently (today) released 7.0.1 SDK](https://firebase.google.com/support/release-notes/cpp-relnotes).

Let me know if you continue to run into issues with 7.x or if this answer doesn't solve your problem!

--Patrick

**Comment by moleki** [comment]
Hey, we just tried to upgrade to firebase c++ sdk 7.1.0 but this time it fails to link on iOS

```
Undefined symbols for architecture arm64:
  "firebase::AppDataDir(char const*, bool, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >*)", referenced from:
      firebase::(anonymous namespace)::GetFilename(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in firebase(heartbeat_date_storage_desktop.cc.o)
ld: symbol(s) not found for architecture arm64
```

We are only using the following frameworks :
 - firebase.framework
- firebase_remote_config.framework
- firebase_analytics.framework
